src/renderer/Renderer.js

```diff
@@ -253,7 +253,7 @@ export default class Renderer {
         const gl = this.gl;
         const aspect = this.getAspect();
         const drawMetadata = {
-            zoom: 1 / this._zoom, // Used by zoom expression
+            zoom: 1 / this._zoom / 1024 * gl.drawingBufferHeight / (window.devicePixelRatio || 1), // Used by zoom expression
```

Comment:
Yes, it can be simplified. `1024` is not particularly special, but it is the resolution at which the usage of zoom() won't produce a breaking change, therefore `1024` seems like a common resolution to me.